app/create_file.py

```
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
+
+    if "-d" in argv:
+        index2 = 0
+
+        for _ in range(len(argv) - 1):
+            index2 += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index2:index - 1]
+else:
+    filename = "file.txt"
+
+    if "-d" in argv:
```

After your `if` statement you should add an `else`. In your logic, if there is no `-f` or `-d` argument you can return an error or something like this.
If you don't add an `else` statement your code will fail